AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/Supporting Files/Base.lproj/Main.storyboard

```diff
@@ -2009,8 +1715,6 @@
         </scene>
     </scenes>
     <resources>
-        <image name="Star_Sprite_0.png" width="290" height="56"/>
-        <image name="logo" width="180" height="180"/>
```

Comment:
@Vedant-Mehta39 to verify warnings from SB are indeed removed

Reply:
Those warnings are gone 👍